src/Components/AddressForm.tsx

```diff
@@ -175,7 +175,7 @@ export const AddressForm: React.FC<AddressFormProps> = ({
           id="AddressForm_addressLine2"
           placeholder="Apt, floor, suite, etc."
           title="Address line 2 (optional)"
-          value={value?.addressLine2}
+          value={value?.addressLine2 || ""}
```

Comment:
Pushed a quick fix for the [non-removed value](https://artsyproduct.atlassian.net/browse/EMI-1320) issue.